RetroBar/Utilities/ExplorerMonitor.cs

```diff
@@ -0,0 +1,60 @@
+using ManagedShell.Interop;
+using System;
+using System.Diagnostics;
+using System.Windows.Forms;
+
+namespace RetroBar.Utilities
+{
+    public class ExplorerMonitor : IDisposable
+    {
+        private bool _explorerMonitorisMonitoring;
+        private ExplorerMonitorWindow _explorerMonitorWindow;
+
+        public void ExplorerMonitorStart(WindowManager windowManagerRef)
+        {
+            if (_explorerMonitorisMonitoring) { return; } // Prevent multiple monitors.
+
+            _explorerMonitorisMonitoring = true;
```

Comment:
Instead of maintaining a bool, can we check that `_explorerMonitorWindow` is not null?